src/hallway/feed.clj

```diff
@@ -0,0 +1,29 @@
+(ns hallway.feed
+  (:require
+    [biff.util :as bu]
+    [hallway.util :as u]))
+
+(def xml-doc
+  "<?xml version=\"1.0\" encoding=\"utf-8\"?>\n")
+
+(defn entry [{:keys [title url created author source]}]
+  [:entry
+   [:title title]
+   [:link {:href url}]
+   [:id url]
+   [:updated (u/format-date created)]
+   [:author [:name author]]
```

Comment:
I removed `author` from the top-level and put it in the individual entries, using the discussion submitter's username for the value.